bouncer/templates/annotation.html.jinja2

```diff
@@ -1,8 +1,5 @@
 {% extends "templates/base.html.jinja2" %}
 
-{% set title = _("Hypothesis: Annotation for %(site)s" )|format(site=pretty_url)  %}
```

Comment:
I don't think you want to remove this line, this is the HTML document title and isn't used in the meta tags.